xds/internal/client/client_loadreport.go

```diff
@@ -44,9 +44,11 @@ func (c *Client) ReportLoad(server string, clusterName string, loadStore lrs.Sto
 		cc      *grpc.ClientConn
 		closeCC bool
 	)
+	c.logger.Infof("Starting load report to server: %s", server)
```

Comment:
This doesn't have "lrs" prefix?

Reply:
This is logging with the logger, from the xds_client. It's going to have prefix "xds-client".

This is where things are getting inconsistent. I will need to refactor LRS to support multiple clusters. The logs in this PR are temporary.